op-sync-tester/synctester/backend/config/static.go

```diff
@@ -14,9 +14,19 @@ type SyncTesterEntry struct {
 
 	// ChainID is used to sanity-check we are connected to the right chain,
 	// and never accidentally try to use a different chain for sync tester work.
-	ChainID eth.ChainID `yaml:"chain_id"`
+	Cfg SyncTesterConfig `yaml:"cfg"`
 }
 
+type TargetBlocks struct {
+	Head      uint64 `yaml:"head"`
+	Safe      uint64 `yaml:"safe"`
+	Finalized uint64 `yaml:"finalized"`
+}
+
+type SyncTesterConfig struct {
+	ChainID eth.ChainID   `yaml:"chain_id"`
+	Target  *TargetBlocks `yaml:"target"`
```

Comment:
~May I ask, why do we need to embed the target block to the config? The sync tester will dynamically initialize using sessions.~ We need the way to make the start blocks into
```go
func (s *Service) SyncTesterEndpoint(chainID eth.ChainID) string {
	uuid := uuid.New()
	return fmt.Sprintf("%s/chain/%s/synctest/%s", s.RPC(), chainID, uuid)
}
```